conda/environments/cudf_dev_cuda10.0.yml

```diff
@@ -6,6 +6,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
+  - rapidsai/label/cuda10.0::cupy>=6*
```

Comment:
Shouldn't need to specify the channel here, the labels will handle it for us.

Reply:
updated